trunk/SUAVE/Methods/Aerodynamics/Lifting_Line/Lifting_Line.py

```diff
@@ -97,7 +94,7 @@ def lifting_line(conditions,settings,geometry):
     if n_segments>0:
         c    = np.ones_like(etan) * wing.chords.root
         ageo = np.ones_like(etan) * wing.twists.root 
-        for i_seg in range(n_segments):
```

Comment:
should this still be -1?

Reply:
No - I made a correction in the geometry of the aircraft a while back so that we define a tip segment. previously it was not defined. (this occurred when I was going through the vehicle scripts to fix the control surface definition)